src/main/scala/dynamodb/mapper.scala

```diff
@@ -597,24 +597,17 @@ trait AmazonDynamoDBScalaMapper {
 
     val builder = Seq.newBuilder[T]
 
-    def local(lastKey: Option[DynamoDBKey] = None): Future[Unit] =
-      client.scan(
-        scanRequest.withExclusiveStartKey(lastKey.orNull)
```

Comment:
now that you are using `scanProgressively` the construction of `scanRequest` above is dead.